test/BUILD.gn

```diff
@@ -1109,7 +1109,9 @@ test("brave_browser_tests") {
   }
 
   if (enable_request_otr) {
-    deps += [ "//brave/browser/request_otr:browser_tests" ]
+    if (!is_android) {
+      deps += [ "//brave/browser/request_otr:browser_tests" ]
```

Comment:
Is there any part of this that can be run on android? It would be better to selectively disable some tests than to not run it at all for android? Can we add some android specific browser tests for the UI? cc @SergeyZhukovsky  for help

Reply:
Currently we have a few Java test for UI to just check if the preference exist or is it removed(in case we remove upstream's), nothing more than that. Not sure do we want to check that or not on Android. Regarding the cc tests above, it looks like they can't be used as is as they access tab_strip_model that doesn't exist on Android, so some refactor has to be done at least. We can create a follow up to make that happen imo.